task2.js

```diff
@@ -1,6 +1,16 @@
 const csvPath = './csv/node_mentoring_t1_2_input_example.csv';
 const csv = require('csvtojson');
+const fs = require('fs');
 
 csv().fromFile(csvPath).then((jsonObj) => {
-  console.log(jsonObj);
+//   console.log(jsonObj);
```

Comment:
On the homework, one of the requirements is:

> Do not load all the content of the csvfile into RAM via stream (read/write file content line by line).

But here you are loading the entire CSV into RAM.

Here you will find a example on how to [use streams ](https://github.com/Keyang/node-csvtojson#use-stream)

Reply:
Also, if using stream, don't forget to add the "error" event listener and log any errors to the console

Reply:
I changed the way to load csv file via stream. But I think it is not line by line.. Still have some problems with this part.

Reply:
Don't worry, I'll help you on the meeting